packages/react-scripts/scripts/test.js

```diff
@@ -32,7 +32,7 @@ const createJestConfig = require('../utils/createJestConfig');
 const path = require('path');
 const paths = require('../config/paths');
 argv.push('--config', JSON.stringify(createJestConfig(
-  relativePath => path.resolve(__dirname, '..', relativePath),
+  relativePath => path.posix.resolve(__dirname.replace(/[\\]+/g, path.posix.sep), '..', relativePath),
```

Comment:
`path.posix` doesn't replace `\` with `/` so we have to do it manually.
/cc @gaearon